README.md

```diff
@@ -46,21 +46,48 @@ Features include:
 Installing
 ==========
 
-There will _soon_ be a release of Swarm on Hackage and in a binary form!
-If you are interested in Haskell or functional programming, you can install
-it from source (see below) or, once it is released, with the Cabal tool.
-For instructions on how to install the binary directly, check the releases.
+**NOTE**: Swarm requires a POSIX-style terminal environment that
+supports `terminfo`.  Linux and MacOS should work fine.  On Windows,
+you will need to use something like the Windows Subsystem for Linux.
+For example, see [the comments on this GitHub
+issue](https://github.com/swarm-game/swarm/issues/53) for instructions
+on building from source on Windows.
 
-Installing from Source
-----------------------
+It is recommended that you use a relatively large terminal window
+(*e.g.* 170 columns x 40 rows or larger).  To find out the size of
+your terminal, you can type `stty size`. If it's not big enough, try
+decreasing the font size. You can read about and/or share recommended
+terminal settings in [this GitHub
+issue](https://github.com/swarm-game/swarm/issues/447).
 
-**NOTE**: Swarm requires a POSIX-style environment that supports
-`terminfo`.  On Linux and OSX, the below instructions should work
-natively.  On Windows, see [the comments on this GitHub
-issue](https://github.com/swarm-game/swarm/issues/53) for instructions
-getting it to work under the Windows Subsystem for Linux.
+- [Installing via binaries](#installing-via-binaries)
+- [Installing from Hackage](#installing-from-Hackage)
+- [Installing from source](#installing-from-source)
+
+Installing via binaries
+-----------------------
+
+Coming soon!
```

Comment:
How about:
> You can download the binary and compressed data from the [latest release](https://github.com/swarm-game/swarm/releases). If you want to run the binary simply as `swarm`, you have to put it in your `PATH`:
> ```bash
> chmod +x ./swarm          # make it executable
> echo $PATH | tr ':' '\n'  # choose one of the printed directories
> mv ./swarm /my/bin/directory/
> ```
> Extract the data to your local Swarm folder otherwise the binary will complain:
> ```bash
> mkdir -p ~/.local/share/swarm/
> unzip data.zip -d ~/.local/share/swarm
> ```

If we use our release action, it will technically be ready sooner than the Hackage release. :grinning: